libs/features/mas/test/merch-card.fries.test.html.js

```diff
@@ -0,0 +1,78 @@
+import { html } from '@esm-bundle/chai';
+import { runTests } from '@web/test-runner-mocha';
+import { expect } from '@esm-bundle/chai';
+
+import { mockLana } from './mocks/lana.js';
+import { mockFetch } from './mocks/fetch.js';
+
+import './spectrum.js';
+import '../src/merch-offer.js';
+import '../src/merch-offer-select.js';
+import '../src/merch-quantity-select.js';
+
+import { mockIms } from './mocks/ims.js';
+import { withWcs } from './mocks/wcs.js';
+import mas from './mas.js';
+
+export default async () => {
+    return runTests(async () => {
+        mockIms();
+        mockLana();
+        await mockFetch(withWcs);
+        await mas();
```

Comment:
you don't need to await mas anymore. it is no more `async`.

Reply:
fixed